src/main/java/team/gif/robot/Constants.java

```diff
@@ -45,10 +45,46 @@
         public static final double kRamseteZeta = 0.7;
     }
 
-    public static class shooter {
-        public static final double kP = 0.18;
+    public static class Shooter {
+        public static final double kP = 0.2;
         public static final double kI = 0;
         public static final double kD = 0;
-        public static final double kF = 0.032;
+        public static final double kF = 0.056;
+
+        public static final double RPM_LOW = 2100;
+        public static final double RPM_HIGH = 7500;
+        public static final double RPM_LAUNCHPAD = 8300;
+        public static final double RPM_PIN_SHOT = 6325; //6325 for close shot
```

Comment:
What is a PIN_SHOT? Is this from the HUB? Call it RPM_HUB

Reply:
"Hub" might be confusing, since we're always shooting at the hub. The official name for the wall is "fender", so maybe that's a better name